### PR TITLE
add(poetry support)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### Added

Adds poetry support for managing package dependencies.

During setup of poetry a dependency issue was found between:
sql alchemy and sqlmodels
The previously locked version of sqlalchemy to 1.4 could not resolve sqmodel 0.0.16.

closes. #83

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by henrikstranneheim — APPROVED** [review]
Great, 👍 . I think we also can remove the setup file ( changes also needed then in bumpconfig). I guess I need to merge https://github.com/Clinical-Genomics/genotype-api/pull/96 first.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "genotype-api"
+version = "2.0.7"
```
[the remark this answers is not in the document]
Comment (ChrOertlin) [reply pyproject.toml]: Done

File: requirements-dev.txt
[the remark this answers is not in the document]
Comment (ChrOertlin) [reply requirements-dev.txt:1]: Yes, now with the bumpversion dependecy removed we can remove this file.

**Comment by ChrOertlin** [comment]
# deploy to stage successful
image

**Comment by henrikstranneheim** [comment]
Would be great to also let the actions use poetry to get rid of requirements files.

```
      - name: Install and configure Poetry
        uses: snok/install-poetry@v1
        with:
          virtualenvs-create: false

**Comment by ChrOertlin** [comment]
Will first update pydantic

**Comment by ChrOertlin** [comment]
closing in favor of #126

**Pull request closed without merging**